### PR TITLE
Removed Open Sans from admin CSS and from admin page.

### LINKED ISSUE #4124: General: Remove Open Sans
Core got rid of a few weeks ago, in [#36753-core](https://core.trac.wordpress.org/ticket/36753). Maybe it's time for us to do something similar?

### PR DESCRIPTION
**Pull request by MichaelArestad**
Fixes #4124 .

#### Changes proposed in this Pull Request:
- Removed Open Sans from various files. Would remove Proxima, but it's not enqueued and isn't used on any current UI.

Looks pretty dang good: 

image

CC @jeffgolenski for review

### CODE REVIEW

**Comment by dereksmart** [comment]
Looks good, and one less request to boot!

**Comment by MichaelArestad** [comment]
Found a spacing bug, but might be a core issue. Fixing temporarily.

**Comment by jeffgolenski** [comment]
Definitely looks good. I think I actually prefer it to open sans :-D

File: scss/atoms/_buttons.scss [hunk 1]
```diff
@@ -11,7 +11,9 @@
 	position: relative;
 	padding: em(10px, 13px) em(19px, 13px);
 	color: #efefef;
-	font: 800 0.9285714286em/1 'Open Sans', Helvetica, sans-serif; // 13/14
+	font-weight: bold;
+	font-size: 0.9285714286em; // 13/14
```
Comment (jeffgolenski) [inline scss/atoms/_buttons.scss]: Well, thanks a fun number. :-P

File: scss/atoms/typography/_variables.scss [hunk 2]
```diff
@@ -2,8 +2,8 @@
 // Typography variables
 // ==========================================================================
 
-$normal: 400; // Change these values when using custom fonts
-$bold: 700; // For example, bold could change to 400;
+$normal: normal; // Change these values when using custom fonts
+$bold: bold; // For example, bold could change to 400;
```
Comment (jeffgolenski) [inline scss/atoms/typography/_variables.scss]: Ah, thanks for getting rid of that. Never even see this!

**Pull request merged**